oka/chap2/knock16.py

```diff
@@ -0,0 +1,36 @@
+# ファイルをN分割
+def filesplit(lines, number):
+    ploto = "C:\\Users\\16t215\\Desktop\\"
+    limit = 24 // number
+    if (24 % number != 0):
+        limit += 1
+    for i in range(number):
+        filenumber = int(i)
+        filename = ploto + "split" + str(filenumber) + ".txt"
```

Comment:
```suggestion
        filename = "{0}split{1}.txt".format(ploto, i)
```
上みたいな書き方もできます．
https://pyformat.info/